BepInEx/Contract/Attributes.cs

```diff
@@ -296,6 +295,7 @@ public static IEnumerable<BepInDependency> GetDependencies(Type plugin)
 	/// <summary>
 	/// An exception which is thrown when a plugin's dependencies cannot be found.
 	/// </summary>
+	[Obsolete]
```

Comment:
I'm fine with removing the class entirely. IIRC it was only ever used internally in chainloader and I can't see how any plugins would've used this, since they don't have access to chainloader internals

Could argue that some plugins _may_ have used it but I'm willing to bet otherwise

Reply:
Done.